ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditorSettings.tsx

```diff
@@ -80,23 +80,25 @@ export function TimeSeriesChartOptionsEditorSettings(props: TimeSeriesChartOptio
       </OptionsEditorColumn>
       <OptionsEditorColumn>
         <ThresholdsEditor hideDefault thresholds={value.thresholds} onChange={handleThresholdsChange} />
-        <Button
-          variant="outlined"
-          color="secondary"
-          onClick={() => {
-            onChange(
-              produce(value, (draft: TimeSeriesChartOptions) => {
-                // reset button removes all optional panel options
-                draft.y_axis = undefined;
-                draft.legend = undefined;
-                draft.visual = undefined;
-                draft.thresholds = undefined;
-              })
-            );
-          }}
-        >
-          Reset All Settings
-        </Button>
+        <OptionsEditorGroup title="Reset Settings">
+          <Button
+            variant="outlined"
+            color="secondary"
+            onClick={() => {
+              onChange(
+                produce(value, (draft: TimeSeriesChartOptions) => {
+                  // reset button removes all optional panel options
+                  draft.y_axis = undefined;
+                  draft.legend = undefined;
+                  draft.visual = undefined;
+                  draft.thresholds = undefined;
+                })
+              );
+            }}
+          >
+            Reset To Defaults
```

Comment:
I found "Reset All Settings" super clear, would you be cool with keeping that text?

Reply:
(it might seem redundant with "Reset Settings", but I think my eye tends to skim the headers, and just read the button text, and that text would immediately be clear to me)